docs/ocps_prototype.rst

```diff
@@ -37,7 +37,7 @@ The prototype client (the ``client/client.py`` module) provides some wrapper fun
 Jobs
 ----------------------
 
-Jobs run as Kubernetes Jobs. The container image for each job is the ``lsstsqre/centos:d_latest`` Docker image.
+Jobs run as Kubernetes Jobs. The container image for each job is the ``ghcr.io/lsst/scipipe:d_latest`` Docker image.
```

Comment:
Actually, I don't think this is even true anymore.  It seems to be running a special TSSW build now.

And you'll note that `ghcr.io` does not appear anywhere in this code.

I think it's best to say "The container image for each job is selected by configuration."